scripts/runfabtests.sh

```diff
@@ -20,13 +18,12 @@ declare -i VERBOSE=0
 declare -r bssh="ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=2 -o BatchMode=yes"
 if [ -z "$(which timeout 2> /dev/null)" ]; then
 	# forego timeout
-	declare -r sssh="${bssh}"
-	declare -r lssh="${bssh}"
+	declare SERVER_CMD="eval"
+	declare CLIENT_CMD="eval"
 else
-	declare -r sssh="timeout 60s ${bssh}"
-	declare -r lssh="timeout 90s ${bssh}"# else
+	declare SERVER_CMD="eval timeout 90s"
+	declare CLIENT_CMD="eval timeout 90s"
```

Comment:
Nope, it will fail if we remove eval. I think it is because of the quotes.
```
eg:
timeout 90s /home/jose/workspace/fabtests.install/bin//fi_msg -f sockets -s 127.0.0.1
-> works
but 
timeout 90s '/home/jose/workspace/fabtests.install/bin//fi_msg -f sockets -s 127.0.0.1'
-> will fail.
```